game/src/main/org/apollo/game/release/r317/Release317.java

```diff
@@ -61,12 +61,13 @@
 	/**
 	 * The incoming packet lengths array.
 	 */
-	public static final int[] PACKET_LENGTHS = { 0, 0, 0, 1, -1, 0, 0, 0, 0, 0, // 0
+	public static final int[] PACKET_LENGTHS = {
+			0, 0, 0, 1, -1, 0, 0, 0, 0, 0, // 0
 			0, 0, 0, 0, 8, 0, 6, 2, 2, 0, // 10
 			0, 2, 0, 6, 0, 12, 0, 0, 0, 0, // 20
 			0, 0, 0, 0, 0, 8, 4, 0, 0, 2, // 30
 			2, 6, 0, 6, 0, -1, 0, 0, 0, 0, // 40
-			0, 0, 0, 12, 0, 0, 0, 0, 8, 0, // 50
+			0, 0, 0, 12, 0, 0, 0, 8, 8, 0, // 50
```

Comment:
Can you show the client code that makes this change required? (Preferably a deob)